src/popupbutton.cpp

```diff
@@ -63,6 +63,15 @@ void PopupButton::performLayout(NVGcontext *ctx) {
 
     const Window *parentWindow = window();
 
+	Widget* checkScroll = parent();
+	while (checkScroll != parentWindow){
+		if (VScrollPanel* derived = dynamic_cast<VScrollPanel*>(checkScroll)){
+			mPopup->setParentPanel(derived);
```

Comment:
Rather than introducing a new ``setParentPanel`` call here, why not issue a modified version of the ``mPopup->setAnchorPos()`` call from below which takes into account the scroll panel offset?